reconcile/test/test_utils_terrascript_cloudflare_client.py

```diff
@@ -232,6 +303,42 @@ def test_create_cloudflare_resources_terraform_json(account_config, backend_conf
                     "depends_on": ["cloudflare_zone.domain-com"],
                 }
             },
+            "cloudflare_logpush_ownership_challenge": {
+                "logpush_ownership_challenge_zone": {
+                    "destination_conf": "s3://bucket/logs?region=us-east-1&sse=AES256",
```

Comment:
Nit: Would it be good to extract specific value like this as constant and referenced it in the mock data and expected result? Also to `logpull_options`?

Reply:
In this case both `destination_conf` and `logpull_options` are not exactly constant, they can have different values depending on Logpush destination and dataset fields. It looks like they are constant because I copy pasted them in other places :sweat_smile: 

Using constant would have made more sense if this field was actually a constant and used in business logic when creating logpush terraform resource through `CloudflareLogpushOwnershipChallengeResource`.  

If we look at other fields within `test_create_cloudflare_resources_terraform_json` , we will see lot of string values present in the spec being copy-pasted in the expected results. Is this necessarily a bad thing? IMO no. There's always a trade-off, we could very well end up with whole bunch of constants which will make test readability a bit poor IMO.